GameFighter/Models/Army.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace GameFighter.Models
+{
+    public class Army
+    {
+        public List<Warrior> ArmyMembers { get; protected set; }
+
+        public int CountUnits => ArmyMembers.Count;
+
+        public int CountAliveUnits
+        {
+            get
+            {
+                int alive = 0;
+                foreach (var item in ArmyMembers)
+                {
+                    if (item.IsAlive)
+                    {
+                        alive++;
+                    }
+                }
+                return alive;
+            }
+        }
+
+        public Army()
+        {
+            ArmyMembers = new List<Warrior>();
```

Comment:
You can use default value for property `public List<Warrior> ArmyMembers { get; protected set; } = new List< Warrior >()`